sdk/harambe/core.py

```diff
@@ -449,6 +449,7 @@ async def run(
         harness: WebHarness = playwright_harness,
         evaluator: Optional[ExpressionEvaluator] = None,
         observer: Optional[OutputObserver | List[OutputObserver]] = None,
+        goto_error_cb: Optional[Callable[..., Awaitable[None]]] = None,
```

Comment:
```suggestion
        goto_error_cb: Callable[..., Awaitable[None]] = default_goto_error_cb,
```

Reply:
Make sure to type the input to the CB, Judging by the cb it should be Callable[[str, int], ...